commands/FBAccessibilityCommands.py

```diff
@@ -86,6 +86,15 @@ def accessibilityLabel(view):
   #using Apple private API to get real value of accessibility string for element.
   return fb.evaluateExpressionValue('(id)[%s accessibilityAttributeValue:%i]' % (view, ACCESSIBILITY_LABEL_KEY), False)
 
+def accessibilityElements(view):
+  a11yElements = fb.evaluateExpression('(id)[%s accessibilityElements]' % view, False)
```

Comment:
This method is only publicly available in iOS 8 and later. Do you know if it's available on iOS 7.x? Ideally we should still support iOS 7.x, or at least should handle it gracefully.